mysql/util_test.go

```diff
@@ -51,3 +55,93 @@ func TestFormatBinaryTime(t *testing.T) {
 		require.Equal(t, test.Expect, string(got), "test case %v", test.Data)
 	}
 }
+
+// mysql driver parse binary datetime
+func parseBinaryDateTime(num uint64, data []byte, loc *time.Location) (time.Time, error) {
```

Comment:
Sorry I don't mean I want to add this test-only function. I want to test `FormatBinaryDateTime` and `toBinaryDateTime`. The output of `FormatBinaryDateTime` is in fact a formatted string of date time, so you can use `time.Parse` to convert the string into `time.Time`, and then use `toBinaryDateTime` to convert `time.Time` to binary representation of date time. Now check it with the input of `FormatBinaryDateTime`